wear/src/main/java/chat/rocket/android/account/ui/AccountFragment.kt

```diff
@@ -0,0 +1,85 @@
+package chat.rocket.android.account.ui
+
+import android.app.Fragment
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.core.view.isVisible
+import chat.rocket.android.R
+import chat.rocket.android.account.presentation.AccountPresenter
+import chat.rocket.android.account.presentation.AccountView
+import chat.rocket.android.util.DrawableHelper
+import chat.rocket.android.util.showToast
+import chat.rocket.android.util.ui
+import chat.rocket.common.model.UserStatus
+import dagger.android.AndroidInjection
+import kotlinx.android.synthetic.main.fragment_account.*
+import javax.inject.Inject
+
+class AccountFragment : Fragment(), AccountView {
+
+    @Inject
+    lateinit var presenter: AccountPresenter
+
+    companion object {
+        fun newInstance() = AccountFragment()
+    }
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        AndroidInjection.inject(this)
+        super.onCreate(savedInstanceState)
+    }
+
+    override fun onCreateView(
+        inflater: LayoutInflater?,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View = inflater!!.inflate(R.layout.fragment_account, container, false)
```

Comment:
Avoid the `!!` here. You can use our extension function from the `util` module instead: `container?.inflate(R.layout.fragment_account)`